microsoft-edge/extensions-chromium/landing/index.yml

```diff
@@ -47,13 +47,13 @@ landingContent:
           - text: Extension concepts and architecture
             url: ../getting-started/index.md
 
-          - text: Create an extension tutorial, part 1
+          - text: "Tutorial part 1: Create a simple extension"
```

Comment:
abandon "tutorial" wording; retitle as:
"Part 1 sample: Display an image in a pop-up"
"Part 2 sample: JavaScript to insert an image in the webpage"

'tutorial' is misleading/ non-advised.  The articles tell the user not to do the "steps" but instead clone the repo, install demo, run demo, and here's how the demo works.  Here are the high-level or theoretical "steps" that were used to create the demo - not steps that we're directing you/user to actually do now.

The articles would read more smoothly if we abandon "tutorial" and "do this" presentation, and word appropriately for what we actually want/intend/advise users to do: clone repo, install Part 1 sample, run Part 1 sample, understand Part 1 sample's code.  see good framing in:
https://github.com/mikehoffms/MicrosoftEdge-Extensions/blob/user/mikehoffms/tut-part-1/README.md

Reply:
noted alt titles in hidden comment below h1 in part1&2 .md

Reply:
might close, b/c requires later reconsidering of:
* "tutorial"
* "do this" wording
* part 2 article is expressed in terms of adding to part 1 code, even though part 2 code is completed
* changing the titles.
* making part 2 worded more standalone, even though, granted, part 2 sample code is mostly a superset added to part 1 sample code.
* stop describing sample 2 as an extension of sample 1 that you create by modifying sample 1.  they are different; two indep samples.  although they do have common aspects, for which, see sample 1's article.  understanding sample 1 is prereq for studying sample 2.

Reply:
shifted focus in titles and intro away from tutorial & "do this coding action" to "here's how the sample works" eg removed "in this tut you write code and create dirs" where possible.